packages/client/src/views/RegisterForm/ApplicationForm.tsx

```diff
@@ -23,29 +23,22 @@ import {
 import { getRegisterForm } from '@opencrvs/client/src/forms/register/application-selectors'
 import { IStoreState } from '@opencrvs/client/src/store'
 import { connect } from 'react-redux'
-import { Event, IForm } from '@client/forms'
-import { IApplication } from '@client/applications'
+import { Event } from '@client/forms'
 
 const pageRoute: { [key in Event]: string } = {
   birth: DRAFT_BIRTH_PARENT_FORM_PAGE_GROUP,
   death: DRAFT_DEATH_FORM_PAGE_GROUP
 }
-export class ApplicationFormView extends React.Component<FullProps> {
```

Comment:
The props for this component should only be the ones returned from `mapStateToProps`  that is `IFormProps` and the `RouteComponentProps`, not `FullProps` .

Reply:
Are

```
application: IApplication
  registerForm: IForm
```

not used at all in this component or its descendants?

Reply:
My concern is that they might be used in RegisterForm and just passed through

Reply:
Yes, they are used in RegisterForm and they are being passed through from ApplicationForm without any problem.